custom/icds_reports/utils/aggregation_helpers/distributed/agg_gov_dashboard.py

```diff
@@ -180,15 +161,17 @@ def update_queries(self):
         FROM temp_gov_dashboard ut
         WHERE agg_gov.awc_id = ut.awc_id;
         """.format(
-            tablename=self.tablename,
+            tmp_tablename=self.temporary_tablename,
         ), {}
 
         yield """
         DROP TABLE temp_gov_dashboard;
         """, {}
 
+        yield f"""
+                    INSERT INTO "{self.staging_tablename}" SELECT * from "{self.temporary_tablename}";
```

Comment:
what is the point of `temporary_table` it seems like it used to insert directly into `staging_table` and now instead it inserts into a table that is otherwise identical and then adds a step to copy

Reply:
you mean to say directly use the Staging table?. I think for that  We would need to change the base class structure alot it creates the staging table in update step which comes after this in parent table.